plutus-use-cases/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, haskell }:
+let
+  puc-scripts-invoker = haskell.packages.plutus-use-cases.components.exes.plutus-use-cases-scripts;
+
+  generated-puc-scripts-output = pkgs.runCommand "plutus-use-cases-scripts-output" { } ''
+    mkdir -p $out/scripts
+    mkdir -p $out/transactions
+    ln -s ${haskell.packages.plutus-use-cases.src}/scripts/protocol-parameters.json protocol-parameters.json
```

Comment:
You can just pick up the file from the source tree, no need to do this roundabout thing. Something like `${builtins.readFile ./scripts/protocol-paramters.json}`

Reply:
Not a big deal, though.